R/pr.R

```diff
@@ -213,10 +224,6 @@ pr_init <- function(branch) {
       ui_bullets(c("v" = "Pulling changes from {.val {remref}}."))
       git_pull(remref = remref, verbose = FALSE)
     }
-  } else {
-    ui_bullets(c(
```

Comment:
Already warned that we can't do this above

Reply:
We didn't actually warn about *this*, i.e. that we can't make sure the base branch is current. So we could add something around line 175. Or not.

Reply:
Ok, tweaked that message a little bit.